### PR TITLE
Update netty

### PR DESCRIPTION
**Pull request by kannanjgithub**

### CODE REVIEW

**Comment by kannanjgithub** [comment]
Created a PR from the fork provided by nicholashagen in https://github.com/grpc/grpc-java/issues/11150.

File: MODULE.bazel [hunk 1]
```diff
@@ -22,20 +22,20 @@ IO_GRPC_GRPC_JAVA_ARTIFACTS = [
...
+    "io.netty:netty-codec-http2:4.1.109.Final",
+    "io.netty:netty-codec-http:4.1.109.Final",
+    "io.netty:netty-codec-socks:4.1.109.Final",
+    "io.netty:netty-codec:4.1.109.Final",
+    "io.netty:netty-common:4.1.109.Final",
+    "io.netty:netty-handler-proxy:4.1.109.Final",
+    "io.netty:netty-handler:4.1.109.Final",
+    "io.netty:netty-resolver:4.1.109.Final",
     "io.netty:netty-tcnative-boringssl-static:2.0.61.Final",
     "io.netty:netty-tcnative-classes:2.0.61.Final",
```
Comment (Wong112t) [inline MODULE.bazel:35]: tcnative should also be bumped to version 2.0.65 to match the bom for Netty 4.1.109

Comment (kannanjgithub) [reply MODULE.bazel:35]: Netty upgrade requires running more tests. We are doing it in https://github.com/grpc/grpc-java/pull/11273.

**Comment by kannanjgithub** [comment]
Netty upgrade requires more exhaustive testing.  We will be handling it and work is in progress in https://github.com/grpc/grpc-java/pull/11273

**Pull request closed without merging**